lib/ff_dpdk_if.c

```diff
@@ -1395,6 +1412,9 @@ ff_rss_check(void *softc, uint32_t saddr, uint32_t daddr,
 
     struct ff_dpdk_if_context *ctx = ff_veth_softc_to_hostc(softc);
     uint16_t reta_size = rss_reta_size[ctx->port_id];
+    uint16_t queueid = qconf->tx_queue_id[ctx->port_id];
+    struct ff_port_cfg *pconf = &ff_global_cfg.dpdk.port_cfgs[ctx->port_id];
```

Comment:
You'd better not use `ff_global_cfg` here, use `lcore_conf` instead, save `pconf->nb_lcores` into `lcore_conf`.Because `lcore_conf` is optimized by cpu cache line.

Reply:
maybe it is enough to move the port related fields to the start position in `struct ff_config`. I think putting these fields to `lcore_conf` is ugly.

Reply:
and we can also remove the `nb_procs` and `proc_lcores` in `struct lcore_conf`

Reply:
`ff_global_cfg` related is only used for initialization, and `lcore_conf` is used for runtime.